content/en/integrations/guide/collect-sql-server-custom-metrics.md

```diff
@@ -4,15 +4,15 @@ kind: guide
 aliases:
   - /integrations/faq/how-to-collect-metrics-with-sql-stored-procedure/
 further_reading:
-- link: "https://www.datadoghq.com/blog/sql-server-metrics/#create-a-stored-procedure-to-generate-and-collect-metrics"
+- link: "https://www.datadoghq.com/blog/sql-server-metrics"
```

Comment:
since this blog post still references the stored procedure method of collecting metrics do you think it makes sense to also add a note here (or somewhere else in this doc) that the `Create a stored procedure to generate and collect metrics` is deprecated & will be removed in a future agent release

Reply:
So there's a [Trello ticket](https://trello.com/c/78weT9H2/2529-edits-to-blog-post-sql-server-metrics) for the writing team to update the blog post but I have no idea when that will get done. Do you think I should just remove the blog post link for now?

Reply:
I'll just remove the link, it's basically duplicating what's in the help article anyway